src/Symfony/Component/Security/Http/Tests/RememberMe/PersistentRememberMeHandlerTest.php

```diff
@@ -126,17 +126,7 @@ public function testConsumeRememberMeCookieValidByValidatorWithoutUpdate()
         $handler->consumeRememberMeCookie($rememberMeDetails);
 
         // assert that the cookie has been updated with a new base64 encoded token value
```

Comment:
can you please update this comment?

Reply:
Totally, I seen it but totally forgot to remove, apologies.